packages/socket-mode/src/SocketModeClient.ts

```diff
@@ -398,6 +398,37 @@ export class SocketModeClient extends EventEmitter {
       }
     });
   }
+
+  /**
+   * Removes secrets and tokens from socket request and response objects
+   * before logging.
+   * @param body - the object with values for redaction.
+   * @returns the same object with redacted values.
+   */
+  private static redact(body?: Record<string, unknown>): Record<string, unknown> | unknown[] | undefined {
```

Comment:
I wanted this to work... Found that [recursive types](https://github.com/slackapi/node-slack-sdk/pull/1832#discussion_r1653545506) sometimes won't work? It might be caused by a certain version of TS cause it seemed to have worked in some earlier version but not in this version 😢